docs/source/user/extensions.md

```diff
@@ -23,45 +23,62 @@ brew install node
 ## Installing Extensions
 
 The base JupyterLab application includes a core set of extensions, which provide
-the features described in this User Guide (Notebook, Terminal, Text Editor,
-etc.)  New extensions can be installed into the application using the command:
+the features described in this User Guide (Notebook, Terminal, Text Editor, etc.)
+You can install new extensions into the application using the command:
 
 ```
 jupyter labextension install <foo>
 ```
 
-Where `<foo>` is a valid JupyterLab extension specifier.  This specifier is
-defined by the extension author in their installation instructions. We are
-encouraging extension authors to add the `jupyterlab-extensions` GitHub topic to
-any repository with a JupyterLab extension. You can see a list of extensions by
-[searching Github for the juputerlab-extensions
-topic](https://github.com/search?utf8=%E2%9C%93&q=topic%3Ajupyterlab-extensions&type=Repositories)
+where `<foo>` is the name of a valid JupyterLab extension on
+[npm](https://www.npmjs.com). If the extension is not published there,
+`<foo>` can also refer to a local directory containing the extension.
+
+We encourage extension authors to add the `jupyterlab-extensions` GitHub topic to
+any repository with a JupyterLab extension to facilitate discovery.
+You can see a list of extensions by searching Github for the
+[jupyterlab-extensions](https://github.com/search?utf8=%E2%9C%93&q=topic%3Ajupyterlab-extensions&type=Repositories)
+topic.
 
-List the currently installed extensions by running the command:
+You can list the currently installed extensions by running the command:
 
 ```
 jupyter labextension list
 ```
 
-Uninstalled an extension by running the command:
+Uninstall an extension by running the command:
 
 ```
 jupyter labextension uninstall <bar>
 ```
 
-Where `<bar>` is the name of the extension, as printed in the extension list.
-Core extensions can also be uninstalled this way (and can later be
+where `<bar>` is the name of the extension, as printed in the extension list.
+You can also uninstall core extensions uninstalled this way (which can later be
 re-installed).
 
-Disable an extension (without unistalilng it) by running the command:
+Installing and uninstalling extensions can take some time, as they are
+downloaded, bundled with the core extensions, and the whole application is rebuilt.
+If you are installing/uninstalling several extensions, you may want to defer
+rebuilding the application by including the flag `--no-build` in the
+install/uninstall step. Once you are ready to rebuild, you can run the command:
```

Comment:
We can also install/uninstall more than one extension at the same time.  Might be worth mentioning above.